relays/ethereum/src/ethereum_client.rs

```diff
@@ -223,8 +216,12 @@ pub async fn best_substrate_block(
 	let (encoded_call, call_decoder) = bridge_contract::functions::best_known_header::call();
 	let call_request = bail_on_arg_error!(
 		to_value(CallRequest {
-			to: Some(contract_address),
+			to: contract_address,
 			data: Some(encoded_call.into()),
+			from: None,
```

Comment:
We should consider adding https://github.com/colin-kiegel/rust-derive-builder to `CallRequest` in web3 crate.